kivy/uix/camera.py

```diff
@@ -78,6 +78,14 @@ class Camera(Image):
     :attr:`resolution` is a :class:`~kivy.properties.ListProperty` and defaults
     to [-1, -1].
     '''
+    
+    apiID = NumericProperty(0)
+    '''Preferred video capture API ID to use when invoking the camera. If you are using
+    0, the API will be the default one::
+
+    :attr:`apiID` is a :class:`~kivy.properties.NumericProperty` and defaults
+    to 0.
```

Comment:
Hi @DarkSparkAg !

Here we need to add a `.. versionadded` snippet, so users can immediately know when this feature has been added.